src/app/pages/ArticlePage/PagePromoSections/TopStoriesSection/index.test.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { ServiceContextProvider } from '#app/contexts/ServiceContext';
+import { ToggleContextProvider } from '#contexts/ToggleContext';
+import * as clickTracking from '#hooks/useClickTrackerHandler';
+import * as viewTracking from '#hooks/useViewTracker';
+import TopStories from '.';
+import { topStoriesList, topStoriesSingleItem } from './fixture';
+
+// eslint-disable-next-line react/prop-types
+const TopStoriesComponent = ({ fixtureData, service = 'mundo' }) => (
+  <ServiceContextProvider service={service}>
+    <ToggleContextProvider>
+      <TopStories content={fixtureData} />
+    </ToggleContextProvider>
+  </ServiceContextProvider>
+);
+
+describe('Optimo Related Content Promo', () => {
+  it('should return null if no data is passed', () => {
+    const { container } = render(<TopStoriesComponent fixtureData={[]} />);
```

Comment:
using `screen` is the recommended way to do dom assertions https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Reply:
oh... I did not know that, thanks for sharing this with me. I will try to fix this soon

Reply:
Done